src/api/users/index.ts

```diff
@@ -0,0 +1,40 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import API from 'api/base';
+import toast from 'react-hot-toast';
+import { UserParams } from './index.types';
+
+export const deleteUser = async (id: string) => {
```

Comment:
this is without createAsyncThunk. but in general everything is good i think.

Reply:
i don't use it in redux